refinery/core/utils.py

```diff
@@ -827,3 +827,24 @@ def create_delete_response_message(del_response):
             # Fix for multiple messages displaying
             messages.set_level(request, messages.ERROR)
             create_delete_response_message(delete_response)
+
+
+def move_obj_to_front(obj_arr, prop_key, prop_value):
```

Comment:
Can we rename `prop_key` and `prop_value` to something like `key_to_match`  & `value_to_match`??